### PR TITLE
Fix countdown timer not updating

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Fixed countdown timer not updating their periods after a change of state in the motion

Resolves DEV-357

### CODE REVIEW

File: src/modules/dashboard/components/ActionsPage/CountDownTimer/CountDownTimer.tsx [hunk 1]
```diff
@@ -143,6 +143,12 @@ const CountDownTimer = ({
     walletAddress,
   ]);
 
+  useEffect(() => {
+    if (state && data) {
+      refetch();
+    }
+  }, [state]);
```
Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/CountDownTimer/CountDownTimer.tsx:150]: This needs to have both `data` and `refetch` as dependencies! Please don't ignore `eslint` rules

Comment (rdig) [inline src/modules/dashboard/components/ActionsPage/CountDownTimer/CountDownTimer.tsx:150]: So this is technically working, however, when the state changes, there is a brief flash where the time is still shows wrong (adding staking + voting time)

Here's an example:
bug-vr-counter-wrong-time-flash

In my example, the timeout is set to 1h. As you can see when the state changes to Reveal, for a brief second it shows as 2h (voting time + reveal time), then, after the `refetch()` call finishes, it shows the proper time.

**Review by rdig — CHANGES_REQUESTED** [review]
This is almost good. It shows the correct time, but when the motion state changes, there  is a brief flash where the time shown is wrong.

While locally this is not a big issue, on production, where everything is slow, this will be more visible.

**Review by chinins — APPROVED** [review]
Looks good to me

**Comment by alicjakujawa** [comment]
@ArmandoGraterol @rdig what is the state of this?

**Comment by rdig** [comment]
Why am I tagged in this ?

**Comment by rdig** [comment]
@alicjakujawa why did you tag me for this ?

**Comment by alicjakujawa** [comment]
> @alicjakujawa why did you tag me for this ?

Because you are as "pending reviewer"

**Comment by rdig** [comment]
I mean... if it's ready I'll gladly review it, but I have no idea what the state of this is

**Comment by ArmandoGraterol** [comment]
@alicjakujawa @rdig This is ready for review! :)

**Review by rdig — CHANGES_REQUESTED** [review]
So after staking (but not so much that the state changes -- imagine just staking 3 out of 10), the counter gets reset.

This is incorrect, since the counter just counts down until the state changes.

Most likely to fix this you'll need to modify the resolver.

**Review by alicjakujawa — CHANGES_REQUESTED** [review]
Unfortunately im still experiencing this bug. When im vote and it will go to reveal state, im seeing voting state time + reveal state time untill i will refresh page.

**Review by alicjakujawa — APPROVED** [review]

**Review by chinins — APPROVED** [review]
Seems to be working properly!

**Pull request merged**